pywinauto/controls/HwndWrapper.py

```diff
@@ -461,6 +462,33 @@ def send_message(self, message, wparam = 0, lparam = 0):
     # Non PEP-8 alias
     SendMessage = send_message
 
+    # -----------------------------------------------------------
+    def send_chars(self, message):
+        """Send a string to the control and wait for it to return"""
+        res = []
+
+        mspl = message.split("{TAB}{ENTER 2}")
+        ind = 0
+        for substr in mspl:
+            ind = ind + 1
+
+            for c in substr:
+                if c in SendKeysCtypes.MODIFIERS.keys():
+                    # Shift, Ctrl, Menu
+                    res.append(-1)
+                else:
+                    # Usual character
+                    res.append(win32api.SendMessage(self.handle, win32con.WM_CHAR, ord(c), 0))
+
+            if ind != len(mspl):
+                # "{TAB}{ENTER 2}"
+                res.append(-2)
+
+        return res
+
+    # Non PEP-8 alias
+    SendChars = send_chars
```

Comment:
Old-style alias is not necessary for new methods. Please leave `send_chars` only.